df_to_azure/export.py

```diff
@@ -2,6 +2,7 @@
 import os
 import sys
 from datetime import datetime
+from io import BytesIO
 from typing import Union
 
 import pandas as pd
```

Comment:
Waarom import je pandas hier helemaal en twee regels hieronder losse onderdelen? Dat is dubbel toch?

Reply:
En je geeft in de readme aan alleen de benodigde functie te gebruiken uit een package, dat klopt dan ook niet meer?

Reply:
Mijn fout, ik las dit in de `__init__.py`, maar wel die van de tests.